src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -317,77 +431,128 @@ export default {
       this.isPending = true
       try {
         if (limits.accountRole == null) {
-          // managelimitbuilder somehow doesnt accept
-          // opts.accountRole NULL value
+          // managelimitbuilder doesnt accept opts.accountRole NULL value
           delete limits.accountRole
         }
-        let accountID
-        // managelimitbuilder somehow doesnt accept opts.accountId NULL value
-        if (limits.accountId) {
-          accountID = limits.accountId
+
+        if (limits.accountID == null) {
+          // managelimitbuilder doesnt accept opts.accountID NULL value
+          delete limits.accountID
```

Comment:
do not modify input parameters, make a deep copy (with lodash’s deepClone, f.e.) and modify it instead